src/__tests__/datasource.test.ts

```diff
@@ -650,8 +650,11 @@ describe('Datasource Query', () => {
           activeAtTimeRange: false,
           columns: [''],
           advancedFilter: '',
-          withAggregate: false,
-          property: [],
+          aggregate: {
```

Comment:
pretty sure this is not supposed to be here.
aggregate is `undefined` for all legacy dashboards